lms/djangoapps/discussion/django_comment_client/base/tests.py

```diff
@@ -1451,7 +1451,6 @@ def setUpClass(cls):
     @classmethod
     def setUpTestData(cls):
         super().setUpTestData()
-        cls.course = CourseFactory.create()
```

Comment:
Support for assigning objects which don’t support creating deep copies with copy.deepcopy() to class attributes in TestCase.setUpTestData() is removed.

Reply:
`cls.course` attribute is already being initiated in `setUpClass` function above so we don't need to do any change.